documentation/man/features/build.rst

```diff
@@ -51,6 +54,18 @@ OPTIONS
   This option allows the user to enable ``ccache`` usage during compilation
   tasks, which should improve compile times in subsequent compilations. 
 
+-w, \--warnings=(1 | 2 | 3 | 12 | 13 | 23 | 123):
+  This can be used to enable compilation warnings accordingly. Please check the
+  kernel's ``make help`` for more info.
+
+-s, \--save-log-to=path:
```

Comment:
Do we need `=`? I think it would be better if we didn't need it.

Reply:
we actually don't use `=`, this was a typo :)